src/components/geolocation/GeolocationDirective.js

```diff
@@ -241,16 +241,20 @@ goog.require('ga_throttle_service');
           elts.removeClass(naClass);
           var msgId;
           switch (error.code) {
-            case error.PERMISSION_DENIED:
+            // PERMISSION_DENIED
+            case 1:
               msgId = 'geoloc_permission_denied';
               break;
-            case error.POSITION_UNAVAILABLE:
+            // POSITION_UNAVAILABLE
+            case 2:
               msgId = 'geoloc_pos_unavailable';
               break;
-            case error.TIMEOUT:
+            // TIMEOUT
+            case 3:
               msgId = 'geoloc_time_out';
               break;
-            case error.UNKNOWN_ERROR:
+            // UNKNOWN_ERROR
+            case 4:
```

Comment:
According to the [spec](https://developer.mozilla.org/en-US/docs/Web/API/PositionError) there's no code number `4` (but it could be implemented in some browser).

And I'll suggest to add a `default` branch, something like:
```js
...
// UNKNOWN_ERROR
case 4:
default:
  msgId = 'geoloc_unknown';
  break;
};
```

Reply:
Good call, thanks